File: 03- Subprogramas y recursividad/subprogramas/java/TestOperaciones.java
/**
 * Descripción: Uso de métodos y sobre carga de métodos.
 * Autor: Santiago Nicolás De la mora Núñez
 * Fecha: 12/02/2026
 */

public class TestOperaciones {
    public static void main(String[] args) {
        int numero1 = 0, numero2 = 0, suma = 0;
        Operaciones.saludar();
        numero1 = Operaciones.solicitarValor();
        numero2 = Operaciones.solicitarValor();
        suma = Operaciones.sumar(numero1, numero2);
        Operaciones.imprimir(suma);
    }
}

File: 03- Subprogramas y recursividad/subprogramas/python/test_operaciones.py
"""
Descripción: Uso de funciones.
Autor: Santiago Nicolás De la mora Núñez
Fecha: 31/03/2026
"""

import operaciones

def main():
    numero1 = 0
    numero2 = 0
    suma = 0

    operaciones.saludar()
    numero1 = operaciones.solicitarValor()
    numero2 = operaciones.solicitarValor()
    suma = operaciones.sumar(numero1, numero2)
    operaciones.imprimir(suma)

if __name__ == '__main__':
    main()